### ISSUE TITLE
Fix invalid aria-label usage on .md-loading-spinner__container element

### ISSUE THREAD

**Issue by IngSchei** [miljodir/md-components#315]
Storybook´s a11y test has identified that the .md-loading-spinner__container element uses an aria-label attribute on a `<div> `element that lacks a valid role attribute. This is not compliant with accessibility standards because aria-label is not supported on a `<div>` without a valid role.

<img width="1010" alt="Image" src="https://github.com/user-attachments/assets/3314ebad-eb66-49c4-b68c-3214274393ad" />
